lib/data/datasources/question_service.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:job_interview_practice/data/models/question.dart';
+
+class QuestionService extends QuestionModel{
```

Comment:
Why do you have a service extending a model?

Reply:
I'm not understand completly. 
I'll serch inheritance and constructor.